js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
+const pushButton = document.querySelector(".first-task button");
+const outArea = document.querySelector(".first-task .out-area");
+const symbol = " $ "; // output symbol
+
+function outputLine(symbolCount) {
+  outArea.innerHTML += `${symbol.repeat(symbolCount)} <br>`;
+}
+
+pushButton.addEventListener("click", () => {
+  outArea.innerHTML = "";
+  let number = Math.round(Number(input.value));
```

Comment:
Please, check how to rename 'number' variable. It is not clear what mean this number.
Also Number() is redundant and it is better to use literal syntax and instead you cant use operator '+'
Math.round(+input.value)

Reply:
done